mtags/src/main/scala/scala/meta/internal/pc/CompletionProvider.scala

```diff
@@ -369,9 +369,22 @@ class CompletionProvider(
       val isTypeMember = kind == CompletionListKind.Type
       params.checkCanceled()
       val matchingResults = completions.matchingResults { entered => name =>
-        if (isTypeMember) CompletionFuzzy.matchesSubCharacters(entered, name)
-        else CompletionFuzzy.matches(entered, name)
+        /** @tgodzik presentation compiler bug
```

Comment:
2 things:
- can you link to the bug?
- can you use the `NOTE(tgodizk): ` notation for consistency with the rest of the codebase?

Reply:
No one reported the bug yet since ScalaIDE uses a different method for matching names and I don't think there is anybody else that uses PC as much. I plan to fix it though, so don't want to report it, just create a PR.

I will add the `NOTE`

Reply:
This bug: https://github.com/scala/scala/pull/8193 uff...

Reply:
Let us see if I manage to get this merged - then we can just fix when upgrading to `2.12.9` and `2.13.1`